### PR TITLE
data: Gautam-666

### PR DESCRIPTION
**Pull request by Gautam-666**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5277"></a>

### CODE REVIEW

**Comment by Gautam-666** [comment]
added tags and social icons ..

File: data/Gautam-666.json [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
-    },
-    {
-      "name": "Follow me on Quora",
-      "url": "https://www.quora.com/profile/GAUTAM-RAI-163",
-      "icon": "FaQuora"
-    }
-  ]
+	"name": "Gautam Rai",
+	"type": "personal",
+	"bio": "A aspiring full stack webdeveloper. Guitarist and Travel enthusiast",
```
Comment (priyankarpal) [inline data/Gautam-666.json:4]: Add a space between Web Developer.
``` suggestion
	"bio": "An aspiring full stack web developer. Guitarist and Travel enthusiast",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Comment by Gautam-666** [comment]
Commited the changes

**Review by TRohit20 — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]

**Comment by Gautam-666** [comment]
I am guessing all the changes are committed. If anything has to be changed just let me know

**Review by Atharv181 — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**